### PR TITLE
[varLib tests] Test for interpolatable.py

### PR DESCRIPTION
**Pull request by miguelsousa**
@anthrotype this test requires an additional package. Is this how to set it up?

### CODE REVIEW

File: Tests/varLib/interpolatable_test.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        suffix = '.ttf'
+        ttx_dir = self.get_test_input('master_ttx_interpolatable_ttf')
+
+        self.temp_dir()
+        ttx_paths = self.get_file_list(ttx_dir, '.ttx', 'TestFamily2-')
+        for path in ttx_paths:
+            self.compile_font(path, suffix, self.tempdir)
+
+        ttf_paths = self.get_file_list(self.tempdir, suffix)
+        self.assertIsNone(interpolatable_main(ttf_paths))
```
Comment (anthrotype) [inline Tests/varLib/interpolatable_test.py]: but.. what is this testing? Simply that `main` completes?
you know, I think it's a bit premature to add tests for this module, it's still very much a work in progress, Behdad may wish to complete it before we sanction the current API with tests like these.

Comment (miguelsousa) [inline Tests/varLib/interpolatable_test.py]: It's testing that it completed without errors. 
OK, I'll stash it for later.

**Comment by behdad** [comment]
> but.. what is this testing? Simply that main completes?

That would be useful test still.

> you know, I think it's a bit premature to add tests for this module, it's still very much a work in progress, Behdad may wish to complete it before we sanction the current API with tests like these.

That's definitely true, but so is all of varLib!

**Comment by behdad** [comment]
> before we sanction the current API with tests like these.

FWIW I never intended for `varLib.interpolate_layout` to remain a module. It was just where I put the code and one day fontmake started using it. I understand working in branch and what not, but when something takes a year to finish on and off, branches are not that helpful.

**Comment by miguelsousa** [comment]
Cosimo, can you take what I submitted and make the necessary changes? Thank you.

**Comment by anthrotype** [comment]
ok, some tests

**Comment by anthrotype** [comment]
ok, having some tests is always better than having none :)

**Pull request merged**